tower-limit/Cargo.toml

```diff
@@ -8,28 +8,28 @@ name = "tower-limit"
 #   - README.md
 # - Update CHANGELOG.md.
 # - Create "v0.1.x" git tag.
-version = "0.3.0-alpha.2"
+version = "0.3.0"
 authors = ["Tower Maintainers <team@tower-rs.com>"]
 license = "MIT"
 readme = "README.md"
 repository = "https://github.com/tower-rs/tower"
 homepage = "https://github.com/tower-rs/tower"
-documentation = "https://docs.rs/tower-limit/0.3.0-alpha.2"
+documentation = "https://docs.rs/tower-limit/0.3.0"
 description = """
 Limit maximum request rate to a `Service`.
 """
 categories = ["asynchronous", "network-programming"]
 edition = "2018"
 
 [dependencies]
-futures-core-preview = "=0.3.0-alpha.19"
-tower-service = { version = "=0.3.0-alpha.2", path = "../tower-service" }
-tower-layer = { version = "=0.3.0-alpha.2", path = "../tower-layer" }
-tokio-sync = "=0.2.0-alpha.6"
-tokio-timer = "=0.3.0-alpha.6"
+futures-core = "0.3"
+tower-service = { version = "0.3", path = "../tower-service" }
+tower-layer = { version = "0.3", path = "../tower-layer" }
+tokio-sync = "0.2.0-alpha.6"
```

Comment:
Shouldn't this be just using the `sync` feature of `tokio`?

Reply:
@jonhoo sync currently does not have semaphore exposed. Carl has said he wants to rewrite it before we expose it :man_shrugging:

Reply:
cc @carllerche

Reply:
Ah! Hmm... Do we want a 0.3 that depends on an `alpha`? I guess maybe it doesn't matter too much since it's a relatively small dependency, and isn't publicly exposed?

Reply:
Yeah, its not public, and we can swap it out for futures-intrusive if we need. So I am punting :D